__tests__/commands/install/issue-2142-regression.js

```diff
@@ -0,0 +1,25 @@
+/* @flow */
+import {runInstall} from '../_helpers.js';
+import * as fs from '../../../src/util/fs.js';
+
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 90000;
+
+const request = require('request');
+const assert = require('assert');
+const path = require('path');
+
+beforeEach(request.__resetAuthedRequests);
+// $FlowFixMe
+afterEach(request.__resetAuthedRequests);
+
+// problem occuring due to optional dependency incompatible with os, in this case fsevents
+// if issue exists, this will not fail on OSX
+test.concurrent('issue #2142 regression test',
```

Comment:
I am not 100% sure when we want to split all tests into separate files.
How about moving it into integration.js for now?
Could you also give it a descriptive name so that people don't go to the web searching what issue 2142 was about?